index.js

```diff
@@ -0,0 +1,159 @@
+const soap = require('soap');
+const Joi = require('joi');
+const { promisify: prom } = require('util');
+
+let SoapClient;
+let url = 'https://devtest.dotfit.com/webservices/OrdersService.asmx?WSDL';
+
+module.exports = {
```

Comment:
any reason not to go with a class - especially since you have gets and sets

Reply:
I would think we'd do new `new DotFitApi({ creds, env: 'dev|prod' )`

Reply:
I had a class originally but I'll switch it out and see how it does